### PR TITLE
docs(rxLocalStorage): FRMW-180-Migrate rxLocalStorage to API Docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-180

- [x] Dev LGTM
- [x] Dev LGTM

Migrate `rxLocalStorage` to API Documentation.

### CODE REVIEW

**Comment by ssalinas24** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

File: src/rxLocalStorage/rxLocalStorage.js [hunk 1]
```diff
@@ -15,11 +15,11 @@ angular.module('encore.ui.rxLocalStorage', [])
...
  * @description
- * A simple wrapper for injecting the global variable localStorage
- * for storing values in local storage. This service is similar to angular's
- * $window and $document services.  The API works the same as the W3C's
+ * A simple wrapper for injecting the global variable `localStorage`
+ * for storing values in the browser's local storage object. This service is similar to Angular's
+ * `$window` and `$document` services.  The API works the same as the W3C's
  * specification provided at: http://dev.w3.org/html5/webstorage/#storage-0.
- * Also includes to helper functions for getting and setting objects.
+ * This service also includes helper functions for `getting` and `setting` objects.
```
Comment (CITguy) [inline src/rxLocalStorage/rxLocalStorage.js]: I don't think "getting" and "setting" need to be wrapped in backticks as they are not properties, methods, or code.

Comment (100stacks) [inline src/rxLocalStorage/rxLocalStorage.js]: :+1:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**